controller/config.go

```diff
@@ -924,7 +925,11 @@ func (c Config) AsSpaceConstraints(spaces *[]string) *[]string {
 	}
 
 	for _, c := range []string{c.JujuManagementSpace(), c.JujuHASpace()} {
-		if c != "" {
+		// TODO (hml) 2019-10-29
+		// Ideally we shouldn't have to check for the AlphaSpaceName,
+		// but it's not a real space yet.  Remove once the alpha space
+		// is no longer needed.  Still check for empty.
+		if c != network.AlphaSpaceName && c != "" {
```

Comment:
And then here we don't need to check if they set it to the alpha. If they did, it was deliberate - apply the filter.